mailbox/api/src/main/java/org/apache/james/mailbox/MailboxSession.java

```diff
@@ -26,17 +26,36 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Random;
 
 import org.apache.james.core.User;
 import org.apache.james.mailbox.model.MailboxConstants;
 
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.MoreObjects;
 
 /**
  * Mailbox session.
  */
 public class MailboxSession {
 
+    @VisibleForTesting
+    public static MailboxSession create(String username) {
```

Comment:
this code doesn't belong to production MailboxSession, it's a test utility, @VisibleForTesting is not made for that.

Reply:
mailbox-api test-jar (because that way, we shouldn't have to change anything in maven)